src/app/views/auth-views/registration/registration.component.html

```diff
@@ -1 +1,42 @@
-<p>registration works!</p>
+<div class="d-flex flex-wrap  justify-content-center align-items-center">
+    <div class="card m-t-50" style="width: 22rem;">
+        <div class="card-body">
+            <div class="d-flex flex-wrap  justify-content-center align-items-center">
+                <p style="font-size: medium;">Registration</p>
+            </div>
+            <form>
+                <div class="form-group">
+                    <label for="exampleInputPassword1">Full Name</label>
+                    <input type="password" class="form-control">
+                </div>
+                <div class="form-group">
+                    <label for="exampleInputEmail1">Email </label>
+                    <input type="email" class="form-control" aria-describedby="emailHelp">
```

Comment:
Remove aria-describedby attribute and change for attribute in label tag

Reply:
Resolved.